assets/js/blocks/cart-checkout/shared/use-view-switcher.tsx

```diff
@@ -23,8 +23,47 @@ export const useViewSwitcher = (
 } => {
 	const initialView = views[ 0 ];
 	const [ currentView, setCurrentView ] = useState( initialView );
+	const viewNames = views.map( ( { view } ) => view );
```

Comment:
This line can be moved inside the `useEffect()`, `viewNames` doesn't seem to be used outside of the effect.